src/thread.rs

```diff
@@ -96,7 +96,8 @@ mod test {
             let time_before = time::SteadyTime::now();
```

Comment:
this is from an external crate, and preferred to be retained to avoid potential name confliction with std::time